presentation/cli.py

```
@@ -15,25 +15,24 @@ def main() -> None:
-    
-    # TODO: Get the appropriate strategy using choose_strategy function
-    # TODO: Apply the strategy to calculate the final total
-    # TODO: Display the results (subtotal, strategy used, and final total)
-    
-    print(f"Subtotal: {subtotal:.2f}")
-    print(f"Strategy: {args.strategy}")
-    # TODO: Calculate and print the final total
+    strat = choose_strategy(args.strategy, percent=args.percent, sku=args.sku,
+                            threshold=args.threshold, per_item_off=args.per_item_off,
+                            max_percent=args.max_percent)
+    total = strat.apply(subtotal, items)
```

Ruff (0.13.1)

Local variable `total` is assigned to but never used

Remove assignment to unused variable `total`

(F841)